src/System.Linq.Expressions/ref/System.Linq.Expressions.cs

```diff
@@ -841,3 +841,24 @@ public sealed partial class UnaryExpression : System.Linq.Expressions.Expression
         public System.Linq.Expressions.UnaryExpression Update(System.Linq.Expressions.Expression operand) { return default(System.Linq.Expressions.UnaryExpression); }
     }
 }
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// An interface to represent values of runtime variables.
+    /// </summary>
+    public interface IRuntimeVariables
+    {
+        /// <summary>
+        /// Count of the variables.
+        /// </summary>
+        int Count { get; }
+
+        /// <summary>
+        /// An indexer to get/set the values of the runtime variables.
+        /// </summary>
+        /// <param name="index">An index of the runtime variable.</param>
+        /// <returns>The value of the runtime variable.</returns>
+        object this[int index] { get; set; }
+    }
+}
```

Comment:
Hmm. Curious and annoying. The commit I have here definitely includes the line break at the end. My local copy definitely shows it, and yet the view through github here does not.